fcrepo-kernel-modeshape/src/main/java/org/fcrepo/kernel/modeshape/observer/FedoraEventImpl.java

```diff
@@ -255,5 +253,15 @@ public static FedoraEvent from(final Event event) {
         }
     }
 
+    private static String cleanPath(final Event event) throws RepositoryException {
```

Comment:
I added some method-level comments about the purpose of this `cleanPath` function.